problemsets-mat223/linearalbegra.tex

```diff
@@ -2462,25 +2462,66 @@ \section*{Matrices}
 		\mat{1&-3&0\\0&0&1\\0&0&0}\mat{x\\y\\z}=\vec b.
 	\]
 	\begin{parts}
-		\item If $\vec b=\mat{1\\2\\3}$, is the set of solutions to this system a 
-		point, line, plane, or other?
-		\item If $\vec b=\mat{1\\1\\0}$, is the set of solutions to this system a 
-		point, line, plane, or other?
+		\item If $\vec b=\mat{1\\2\\3}$, is the set of solutions to this system
+			a point, line, plane, or other?
+			\begin{solution}
+				This system has no solutions, since the third equation would be
+				$0=3$, which is impossible. 
+			\end{solution}
+		\item If $\vec b=\mat{1\\1\\0}$, is the set of solutions to this system
+			a point, line, plane, or other?
+			\begin{solution}
+				A line.	The system would be
+				\begin{align*}
+					x - 3y &= 1 \\
+					z &= 1 \\
+					0 &= 0
+				\end{align*}
+				$\vec x=\mat{x\\y\\z}$ satisfies this system if and only if it is
+				on the line $\vec x = t \mat{3\\1\\0} + \mat{1\\0\\1}$.
+			\end{solution}
 	\end{parts}
 
 	\question
-	Let $\vec d_1=\mat{1\\1\\2}$ and
-	$\vec d_2=\mat{-1\\1\\0}$. 
+	Let $\vec d_1=\mat{1\\1\\2}$ and $\vec d_2=\mat{-1\\1\\0}$. 
 	Let $\mathcal P$ be the plane given in vector form by $\vec x=t\vec d_1+s\vec d_2$.
 	Further, suppose $M$ is a matrix so that $M\vec r\in\mathcal P$ for any $\vec r$.
 	\begin{parts}
 		\item How many rows does $M$ have?
+			\begin{solution}
+				Three. It must have three rows in order for $M\vec r$ to be an
+				element of $\R^3$. 
+			\end{solution}
 		\item Find such an $M$.
-		\item Find necessary and sufficient conditions (phrased as equations) for $\vec n$
-			to be a normal vector for $\vec p$.
-		\item Find a matrix $K$ so that solutions
-			to $K\vec x=\vec 0$ are normal vectors for $\mathcal P$.
-			How do $K$ and $M$ relate?
+			\begin{solution}
+				$M=\mat{1&-1&0\\1&1&0\\2&0&0}$ is one possible answer, since if
```

Comment:
Why the third column?

Reply:
Sorry, I don't think I know what you mean. M should have three columns so that its multiplication with column vectors in R^3 is defined. I could put something non-zero in the third column if you prefer. Like for example we could make M the column matrix M=[d_1 d_2 d_2].

I might be missing something obvious though...

Reply:
Did you intend for \vec r to be an element of R^2?

Reply:
Oh, okay! As worded the question really feels like \vec r is in R^3, since everything else about the question is in R^3. I was very confused by your comments. I'm going to amend the question slightly to make it clear that \vec r is a vector in R^2, if that's okay.

Now that I better understand the question you wanted answered, is the matrix M I found "too good"? Like for a general M that does what part 2 wants, all you could say that relates M and K is about their sizes (3x2 vs 2x3), but the ones I found are *transposes* of each other. Maybe that's too special of a relationship?